calendar_resource/README.rst

```diff
@@ -10,11 +10,26 @@ This module adds new features to facilitate resource management with meetings:
 
 * To attach resources to a meeting, go to `Calendar` => Click on a meeting => `Edit` =>
   `Edit` => add any related resources.
-* To edit or add a resource, go to `Settings` => `Resource` => `Resources`.
+* To edit or add a resource, go to `Settings` => `Resource` => `Resources`. Developer mode
+  has to be enabled for this menu to show.
```

Comment:
Yeah I'm not sure exactly why it was placed there originally in resource. Perhaps we should create a menu in the top navbar? Name it 'Resources'.